ParallelRoadTool/Detours/NetManagerDetour.cs

```diff
@@ -348,7 +355,14 @@ private static ushort NodeAtPositionOrNew(ref Randomizer randomizer, NetInfo inf
                         newEndNodeId, startDirection, endDirection,
                         Singleton<SimulationManager>.instance.m_currentBuildIndex + 1,
                         Singleton<SimulationManager>.instance.m_currentBuildIndex, invert);
-                }                
+                }
+                // Left-hand drive revert conditions back
+                if (ParallelRoadTool.Instance.IsLeftHandTraffic)
+                {
+                    invert = !invert;
+                    isReversed = !isReversed;
+                    _isPreviousInvert = invert;
```

Comment:
Shouldn't we revert `ParallelRoadTool.Instance.IsSnappingEnabled = true;` too and setting it to false?